lookup/functions/lookup.js

```diff
@@ -1,46 +1,59 @@
+// pulls nested error codes from response data
+function getErrorLinks(data) {
+  const hasErrorCode = (value) =>
+    value !== null &&
+    typeof value !== 'undefined' &&
+    value.hasOwnProperty('error_code') &&
+    value.error_code !== null;
+
+  const errors = Object.entries(data)
+    .filter(([_, v]) => hasErrorCode(v))
+    .map(([k, v]) => [k, v.error_code]);
+
+  return errors.map(([k, errorCode]) => {
+    return `${k} error: <a href="https://www.twilio.com/docs/api/errors/${errorCode}" target="_blank">${errorCode}</a>`;
```

Comment:
would it be possible to not use single letter variable names? Strongly prefer not doing so for code that's meant for people to be able to expand on or learn from.

Reply:
Absolutely - great call